README.md

```diff
@@ -172,6 +172,46 @@ TID is a 32-bit integer to identify threads created with `wasi_thread_spawn`.
     For example, it can be used to indicate the main thread, which doesn't
     have a TID in the current version of this proposal.
 
+### Process
+
+* A process is a group of threads.
+
+* The main thread starts with a process which only contains
+  the main thread.
+
+* Threads created by a thread in a process using `wasi_thread_spawn`
+  are added to the process.
+
+* When a thread is terminated, it's removed from the process.
+
+### Voluntary thread termination
+
+A thread can terminate itself voluntarily by returning from
+`wasi_thread_start`.
+
+### Changes to WASI `proc_exit`
+
+With this proposal, the `proc_exit` function takes extra responsibility
+to terminate all threads in the process, not only the calling one.
```

Comment:
How about simply "When `proc_exit` is called the entire process is terminated, including all running threads".   I'm not we need the "takes extra responsibility" wording.   

It seems fairly clear from its name that `proc_exit` would do this, so I don't really see it as "extra responsibility", but maybe thats just my reading of it.

Reply:
without wasi-threads, a runtime would simply terminate the calling thread.

when adding wasi-threads support to a runtime, you need to make it terminate other thread too. it would need a significant implementation effort. i feel it's appropriate to call it extra.

Reply:
Sure.   But don't think that is clear enough to simply say: "When proc_exit is called the entire process is terminated, including all running threads".

(BTW, the way I see it `proc_exit` brings down the entire process (that is that the `proc` part means), even without wasi-threads.  It just happens that there was only ever one thread previously.)